src/routes/api/ghes-app-encryption-ctx/re-encrypt-ghes-app-keys.ts

```diff
@@ -88,11 +88,11 @@ export const ReEncryptGitHubServerAppKeysPost = async (req: Request, res: Respon
 				gitHubClientSecret: originGitHubClientSecret
 			}, jiraHost);
 
-			const updatedApp: GitHubServerApp = await GitHubServerApp.findByPk(app.id);
+			const updatedApp: GitHubServerApp | null = await GitHubServerApp.findByPk(app.id);
```

Comment:
I'd prefer to either use "!" here because we know that the app exists.

Reply:
I had tried that initially but unfortunately it doesn't clear the error.

<img width="658" alt="Screenshot 2023-04-03 at 9 58 22 am" src="https://user-images.githubusercontent.com/37155488/229386014-8ca72999-d81c-4665-b512-fc7bf02f683c.png">

The `findByPk` method returns the entry or null:

```
public static findByPk<M extends Model>(
    this: ModelStatic<M>,
    identifier?: Identifier,
    options?: Omit<FindOptions<Attributes<M>>, 'where'>
  ): Promise<M | null>;
```

Reply:
`await GitHubServerApp.findByPk(app.id)!` ?